.github/workflows/comprehensive_tests.yml

```
@@ -0,0 +1,329 @@
+      
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: frontend-build
+          path: telegram-mini-app/dist/
+
+  performance-tests:
+    name: Performance Tests
+    runs-on: ubuntu-latest
+    needs: [backend-tests, frontend-tests]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)